UnusedIO: init of empty list

I am writing tests using `munit.CatsEffectSuite` and using `typelevel-scalafix v0.1.3`.

```scala
import munit.CatsEffectSuite

final class MySuite extends CatsEffectSuite {
  test("foo") {}
}
```

This will result in

```
[error] Caused by: java.lang.UnsupportedOperationException: init of empty list
[error] 	at scala.collection.immutable.Nil$.init(List.scala:666)
[error] 	at scala.collection.immutable.Nil$.init(List.scala:661)
[error] 	at org.typelevel.fix.UnusedIO$$anonfun$checkTree$1.applyOrElse(UnusedIO.scala:90)
[error] 	at org.typelevel.fix.UnusedIO$$anonfun$checkTree$1.applyOrElse(UnusedIO.scala:84)
[error] 	at scala.PartialFunction$Lifted.apply(PartialFunction.scala:338)
[error] 	at scala.PartialFunction$Lifted.apply(PartialFunction.scala:334)
```

--- a/modules/cats-effect/input/src/main/scala/fix/UnusedIOTests.scala
+++ b/modules/cats-effect/input/src/main/scala/fix/UnusedIOTests.scala
@@ -246,6 +246,15 @@ object UnusedIOTests {
     } yield ()
   }
 
+  def unusedEmptyIO = {
+    for {
+      a <- IO.pure(42L)
+      _ = IO {} /* assert: TypelevelUnusedIO.unusedIO
+          ^^^^^
+          This IO expression is not used. */
+    } yield ()
+  }
+
   // TODO: Implement checks using inferred type information
   // This is tricky to do because the inferred types in the
   // call to `.value` are bound in the previous expression.

--- a/modules/cats-effect/rules/src/main/scala/org/typelevel/fix/UnusedIO.scala
+++ b/modules/cats-effect/rules/src/main/scala/org/typelevel/fix/UnusedIO.scala
@@ -87,7 +87,7 @@ class UnusedIO extends SemanticRule("TypelevelUnusedIO") {
     case Term.While(_, body) =>
       checkDiscardedStat(body)
     case Term.Block(stats) =>
-      stats.init.map(checkDiscardedStat).asPatch
+      stats.dropRight(1).map(checkDiscardedStat).asPatch
     case Term.Interpolate(_, _, args) =>
       args.map(checkDiscardedStat).asPatch
     case Term.Try(_, _, Some(finalizer)) =>

